composer/loggers/mlflow_logger.py

```diff
@@ -198,7 +201,7 @@ def log_metrics(self, metrics: Dict[str, Any], step: Optional[int] = None) -> No
         from mlflow import log_metrics
         if self._enabled:
             # Convert all metrics to floats to placate mlflow.
-            metrics = {k: float(v) for k, v in metrics.items()}
+            metrics = {k: float(v) for k, v in metrics.items() if k not in self.ignore_metrics}
```

Comment:
I think you want this to a be a regex, or you're going to have a bad time listing out all the exact metrics you want to ignore.

Reply:
I think glob wildcards is a nice happy medium between exact strings and regex